docs/tutorial/part01.rst

```diff
@@ -2,5 +2,90 @@
 Set up the environment.
 =======================
 
-.. todo::
-   Here will be the tutorial body.
\ No newline at end of file
+The following section will have the steps to set your local environment, so you can begin to work on your
+Django-neomodel implementation. If you want a point to start, we provide a base app to begin the tutorial.
+
+First clone the paradise-papers-django_ repo.
+
+    .. _paradise-papers-django: https://github.com/neo4j-examples/paradise-papers-django
+
+After you download the repo go to the ``start_app`` branch and you will get a working app that will be our start point.
+
+.. [*] Note:
+    If you use the start branch mentioned before you only will get just the front-end of the app.
+
+Set your virtualenv
+===================
+
+First, you need Python and pip install on your local environment. Here is the official download page of python_
+
+    .. _python: https://www.python.org/downloads/
+
+Here is a quick tutorial on how to install pip_
+
+    .. _pip: https://pip.pypa.io/en/stable/installing/#installing-with-get-pip-py
+
+Now we have everything we need to create our virtualenv. On the next part we are going to introduce the basic to get
+your app running; if you want more information check this virtualenv_ tutorial
+
+    .. _virtualenv: https://virtualenv.pypa.io/en/stable/
+
+First go to your app folder on the case of the paradise-papers-django project you need to be on the main folder
+(localPath/ paradise-papers-django) and run the following command to install virtualenv::
+
+    pip install virtualenv
+
+You can use this pip command to check the dependencies installed::

```

Comment:
Here I'm confused, why are you talking about the dependencies before running virtualenv? This will just bring the dependencies installed in their local env,

Reply:
Is just to verify if you install virtualenv. You can use pip freeze outside of the virtual environment to check you current dependencies and look if you already installed it.